OpenEdXMobile/res/values/dimens.xml

```diff
@@ -65,11 +65,16 @@
     <dimen name="login_logo_margin_bottom">15dp</dimen>
     <dimen name="login_social_divider_margin_top">20dp</dimen>
     <dimen name="login_social_divider_margin_bottom">15dp</dimen>
+    <dimen name="divider_with_text_margin">12dp</dimen>
```

Comment:
@mdinino If you didn't think it belonged with the other divider you should have kept it there and said why?